src/computeMatrix.m

```diff
@@ -20,7 +20,7 @@
 function systemDeltauMatrix = computeMatrix( Conec, elements, Nodes, materials, KS, analysisSettings, Uk, Udott, Udotdott, neumdofs, nodalDispDamping ) ;
 
   % computes static tangent matrix
-  [ ~, ~, mats ] = assembler( Conec, elements, Nodes, materials, KS, Uk, Udott, Udotdott, analysisSettings, [0 0 1], nodalDispDamping ) ;
+  [ ~, ~, mats ] = assembler( Conec, elements, Nodes, materials, KS, Uk, Udott, Udotdott, analysisSettings, [0 0 1], nodalDispDamping, analysisSettings.deltaT ) ;
```

Comment:
This line should change in future if we are going to compute aerodynamic damping matrix, as it depends on wind velocity and aerodynamic coefficients.

Reply:
ok